presto-hive/src/test/sql/create-test-hive13.sql

```diff
@@ -82,7 +82,7 @@ SELECT * FROM presto_test_types_textfile
 ;
 
 
--- Parquet is missing TIMESTAMP and BINARY, and for some reason
```

Comment:
We should consider if we want to create a new integration environment for newer versions of Hive or upgrade the Hive 0.13 version.